curriculum/challenges/english/01-responsive-web-design/responsive-web-design-projects/build-a-personal-portfolio-webpage.english.md

```diff
@@ -50,7 +50,7 @@ tests: []
 ## Solution
 <section id='solution'>
 
-```js
+```html
 // solution required
```

Comment:
@ojeytonwilliams Even though this is now HTML, is it a problem that the comment is a JavaScript comment?

Reply:
Good catch, but no, it should not matter. It should just mean this challenge's tests should not run.